xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/config/LiquibaseRunner.java

```diff
@@ -0,0 +1,45 @@
+package com.icthh.xm.commons.domain.event.config;
+
+import com.icthh.xm.commons.domain.event.util.TenantUtil;
+import liquibase.integration.spring.SpringLiquibase;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ResourceLoader;
+
+import javax.sql.DataSource;
+
+@Slf4j
+@Configuration
+@RequiredArgsConstructor
+@EnableConfigurationProperties({LiquibaseProperties.class})
+public class LiquibaseRunner {
```

Comment:
I see here code similarity with `com.icthh.xm.commons.migration.db.tenant.provisioner.TenantDatabaseProvisioner`
is it possible to reuse existing class?